gegelatilib/include/mutator/mutationParameters.h

```diff
@@ -63,6 +55,16 @@ namespace Mutator {
         /// Number of root TPGTeams to maintain when populating the TPGGraph
         size_t nbRoots = 100;
 
+        /// JSon comment
+        inline static const std::string initNbRootsComment =
+            "// Number of root TPGTeams at the initialisation of a "
+            "TPGGraph.\n"
+            "// If 0, if will be init to the number of actions\n"
+            "// \"nbRoots\" : 0, // Default value";
```

Comment:
Not entirely exact from what I understand from learningAgent.h.
`If 0, the number of initial root teams will be the maximum between the number of actions, and the number of surviving roots defined by mutations parameters.`